okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -396,6 +403,55 @@ public OkHttpChannelBuilder connectionSpec(
     return this;
   }
 
+  /**
+   * Sets the connection specification used for secure connections.
+   *
+   * <p>By default a modern, HTTP/2-compatible spec will be used.
+   *
+   * <p>This method is only used when building a secure connection. For plaintext
+   * connection, use {@link #usePlaintext()} instead.
+   *
+   * @param tlsVersions List of tls versions.
+   * @param cipherSuites List of cipher suites.
+   * @param supportsTlsExtensions Support of TLS extensions.
+   */
+  public OkHttpChannelBuilder tlsConnectionSpec(
+          String[] tlsVersions, String[] cipherSuites,
+          boolean supportsTlsExtensions) {
```

Comment:
gRPC really requires the extensions, so I don't think there's much point to have the caller provide it here.

Reply:
make sense, I will remove the `supportsTlsExtensions` from the function